lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
```

Comment:
RE: below

perhaps it would be better to declare class_attributes on the base class, ref: https://coderwall.com/p/xhcmbg/cattr_accessor-vs-class_attribute

```
base.class_attribute :weight
base.class_attribute :authorization
# ...
```